packages/cli-exec/src/exec.js

```diff
@@ -120,7 +120,9 @@ async function* spawn(cmd, args, percy) {
 
     if (proc.stderr) {
       proc.stderr.on('data', (data) => {
-        errorMessage += data.toString();
+        const message = redactSecrets(data.toString())
```

Comment:
I would recommend to run this separately later before sending. Here the data comes in chunks so its not necessary that the chunk with succeed regex match when you have a secret split across multiple chunks. I am not sure if thats possible or not, but we need a confirmation.